pkg/generated/clientset/versioned/typed/workerpodautoscaler/v1alpha1/workerpodautoscaler.go

```diff
@@ -63,6 +64,18 @@ func newWorkerPodAutoScalers(c *K8sV1alpha1Client, namespace string) *workerPodA
 	}
 }
 
+// validate check if the caller has set the right fields.
+func validate(s *v1alpha1.WorkerPodAutoScaler) error {
+	if *s.Spec.MaxReplicas < 1 {
+		return fmt.Errorf("'max' is a required parameter and must be at least 1")
```

Comment:
It might be useful to set max=0 sometimes (like when you want to temporarily disable processing of jobs). Also, we can put the static CRD validations like this one in the OpenAPI spec itself. min >= 0 and max >= 0 makes sense for me.